### PR TITLE
Contract Testing

### PR DESCRIPTION
**Pull request by nadinequinr3**

### CODE REVIEW

File: content/en/tools-corda5/contract-testing/_index.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+| daveName              | daveKey     |
+| eveName               | eveKey      |
+| bankAName             | bankAKey    |
+| bankBName             | bankBKey    |
+| notaryName            | notaryKey   |
+
+### Happy Path Testing
+
+If you expect your test to pass the contract validation, use `assertVerifies`. For example:
+```java
```
Comment (yarivus) [inline content/en/tools-corda5/contract-testing/_index.md:61]: please indent code snippets w/ 4 spaces (or 8)

**Review by yarivus — CHANGES_REQUESTED** [review]

File: content/en/tools-corda5/contract-testing/_index.md (hunk 1 above)
Comment (nadinequinr3) [reply content/en/tools-corda5/contract-testing/_index.md:61]: > please indent code snippets w/ 4 spaces (or 8)

Do you mean the whole code block? Or just particular lines?

Comment (yarivus) [reply content/en/tools-corda5/contract-testing/_index.md:61]: each code block should be relatively indented by 4 spaces instead of 2

Comment (nadinequinr3) [reply content/en/tools-corda5/contract-testing/_index.md:61]: Have updated EACH line (except the first) of both code blocks.

File: content/en/tools-corda5/contract-testing/_index.md [hunk 2]
```diff
@@ -60,33 +60,33 @@ table th:nth-of-type(2) {
 If you expect your test to pass the contract validation, use `assertVerifies`. For example:
 ```java
 @Test
```
Comment (yarivus) [inline content/en/tools-corda5/contract-testing/_index.md:62]: should be on the same column as public (start at the same indent)

File: content/en/tools-corda5/contract-testing/_index.md [hunk 3]
```diff
@@ -60,33 +60,33 @@ table th:nth-of-type(2) {
...
+            .toSignedTransaction();
+        assertVerifies(transaction);
+    }
 ```
 
 ### Negative Path Testing
 
 If you expect the contract validation to reject the transaction, make use of `assertFailsWith`. For example:
 ```java
 @Test
```
Comment (yarivus) [inline content/en/tools-corda5/contract-testing/_index.md:79]: `@Test` should be on the same column as public (start at the same indent)

**Review by yarivus — COMMENTED** [review]
see minor comments

**Review by yarivus — APPROVED** [review]
LGTM

**Review by yarivus — APPROVED** [review]
LGTM

**Review by yarivus — APPROVED** [review]
LGTM

File: content/en/tools-corda5/contract-testing/_index.md [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+title: "Contract Testing"
+project: corda5-tools
+section_menu: corda5-tools
+version: tools
+menu:
+  corda5-tools:
+    weight: 1000
+---
+
+The Contract Testing library enables {{< tooltip >}}CorDapp{{< /tooltip >}} Developers to test smart contracts locally, at an early stage of the development cycle. With this library, you can check that your CorDapp contracts behave as expected before, or after, you write the flows.
```
Comment (paulmoloneyr3) [inline content/en/tools-corda5/contract-testing/_index.md:12]:
```suggestion
The Contract Testing library enables {{< tooltip >}}CorDapp{{< /tooltip >}} developers to test smart contracts locally at an early stage of the development cycle. With this library, you can check that your CorDapp contracts behave as expected before, or after, you write the flows.
```

**Review by paulmoloneyr3 — APPROVED** [review]

**Pull request merged**